### DISCUSSION TITLE
How i set environments in Strapi RC ?

### DISCUSSION THREAD

**Discussion by pedrolaxe** [strapi/strapi#6117] [Legacy Topics]
Hello guys, I installed the new RC version, i note that we no longer have config/ environments, would you like to know how I configure my application for develop / staging / production? .ENV?

**Comment by abdonrd** [comment]
You can see the future documentation here:

https://github.com/strapi/strapi/blob/master/docs/3.0.0-beta.x/concepts/configurations.md

**Reply by lauriejim** [reply]
Thank you @abdonrd for this link.
